src/kernels/kernelConnector.ts

```diff
@@ -425,10 +426,12 @@ export class KernelConnector {
                     }
                 }
             } catch (error) {
-                traceWarning(
-                    `Error occurred while trying to ${currentContext} the kernel, options.disableUI=${options.disableUI}`,
-                    error
-                );
+                if (!isCancellationError(error)) {
+                    traceWarning(
```

Comment:
No point adding logs when its a cancellation error, we just end up with lots of Errors logged in the output along with stack trace when these are not real errors (basically filling the error logs when running simple tests) - thus leading to false positives - i.e. it looks as though there are errors, when infact there aren't any.